stl/inc/flat_set

```diff
@@ -447,9 +443,9 @@ public:
     }
 
 private:
-    _NODISCARD bool _Check_sorted(const_iterator _It, const const_iterator _End) const {
+    _NODISCARD bool _Is_sorted(const_iterator _It, const const_iterator _End) const {

```

Comment:
It might be better to name this function as `_Is_Tsorted`, if `T` means "type-dependent".
`_Tsorted->_Tsorted_tag`
`_Is_sorted->_Is_Tsorted`
`_Msg_not_sorted->_Msg_not_Tsorted`

However I'm not very sure about the original meaning of `T` in `_Tsorted`...